src/textual_fspicker/file_dialog.py

```diff
@@ -182,8 +182,13 @@ def _confirm_file(self, event: Input.Submitted | Button.Pressed) -> None:
         try:
             if chosen.is_dir():
                 if sys.platform == "win32":
-                    if drive := MakePath.of(file_name.value).drive:
```

Comment:
Why is this code being removed to be replaced with a slightly more verbose version?